smart_selects/views.py

```diff
@@ -1,9 +1,7 @@
 from django.db.models import get_model
 from django.http import HttpResponse
-try:
-    import json
-except ImportError:
-    from django.utils import simplejson as json
+from django.utils.encoding import force_text
+import json
```

Comment:
Django 1.4 still supports Python 2.5, which lacks the `json` module. Is there a reason you needed to break compatibility here?